src/test/resources/zoldicatalina/groups-tests.xml

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
You either try to use your groups/classes from teorie/TestNg package with proper changes or you remove this file. Thanks!